stingray/events.py

```diff
@@ -669,50 +800,166 @@ def to_astropy_timeseries(self):
             ts = TimeSeries(data=data, time=times)
         else:
             ts = TimeSeries()
-        ts.meta['gti'] = self.gti
-        ts.meta['mjdref'] = self.mjdref
-        ts.meta['instr'] = self.instr
-        ts.meta['mission'] = self.mission
-        ts.meta['header'] = self.header
+
+        ts.meta.update(self.get_meta_dict())
+
         return ts
 
     @staticmethod
     def from_astropy_timeseries(ts):
-        from astropy.timeseries import TimeSeries
-        from astropy import units as u
-        energy = pi = gti = instr = mission = mjdref = None
-        if 'energy' in ts.colnames:
-            energy = ts['energy']
-        if 'pi' in ts.colnames:
-            pi = ts['pi']
+        """Read the event list from an Astropy TimeSeries.
```

Comment:
Minor docstring nitpick. I think it's worth distinguishing the data type "event list" from the stingray class `EventList`. So perhaps this first sentence should read something like: "Read event list data from an astropy TimeSeries object and store in an `EventList` object."? Similarly use `EventList` as appropriate below to designate the object you're creating